lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -122,6 +122,43 @@ class CategoryRenderer {
    * @param {number} scale
    * @return {!Element}
    */
+  _renderTimelineMetricAudit(audit, scale) {
+    const element = this._dom.createElement('div', [
+      'lh-timeline-metric',
```

Comment:
nit: no need for join()

 \`lh-timeline-metric lh-timeline-metric--${Util.calculateRating(audit.score)}`